start1: [-0.06442719301757813, -0.3482136384521485, 1.334563283935547, -1.0093593572753907, 0.19021361748046875, 4.609612262329102]
start2: [0.09510680874023437, -0.009203884716796876, 1.802427423706055, -0.2439029449951172, 0.19174759826660157, 4.609612262329102]
approach3: [0.10124273188476564, -0.9188544908935548, 1.0630486847900391, -0.14112623232421875, 0.20555342534179688, 4.611146243115234]
reach4: [-0.0015339807861328126, -1.4434759197509766, 0.3558835423828125, 0.04141748122558594, 0.20401944455566406, 4.609612262329102]
grab5: [-0.015339807861328126, -0.8252816629394532, 1.294679783496094, 0.03528155808105469, 0.20555342534179688, 4.640291878051758]
pullback6: [0.027611654150390626, -0.4264466585449219, 1.7855536350585939, -0.07209709694824219, 0.20555342534179688, 4.620350127832031]
turn7: [0.5783107563720703, -0.4233786969726563, 1.9650293870361328, -0.981747703125, 0.20401944455566406, 3.969942274511719]
move8: [0.3067961572265625, -0.696427276904297, 1.2716700717041016, -1.5677283634277346, 0.196349540625, 4.313553970605469]
